src/modules/transfer/components/SelectInputSender.vue

```diff
@@ -43,6 +44,26 @@ props: [
     'modelValue',
     'selectDefault'
   ],
+
+data(){
+  return {
+    filterQuery: "",
+  }   
+},
+
+computed: {
+  filteredAccounts() {
+      const query = this.filterQuery.toLowerCase();
+      if(this.filterQuery == ""){
+        return this.accounts;
+      }
+      return this.accounts.filter((items) =>{
+        return Object.values(items).some((word) =>
+          String(word).toLowerCase().includes(query));
+      });
+    }
+},
+
```

Comment:
why is this in vue2 syntax? it's not wrong, but it makes the code messy

Reply:
noted, will work on it